databricks/koalas/series.py

```diff
@@ -1086,6 +1087,12 @@ def nunique(self, dropna: bool = True, approx: bool = False, rsd: float = 0.05)
 
         >>> ks.Series([1, 2, 3, np.nan]).nunique(dropna=False)
         4
+
+        On big data, we recommend using the approximate algorithm to speed up this function.
+        The result will be very close to the exact unique count.
+
+        >>> ks.Series([1, 2, 3, np.nan]).nunique(approx=True)  # doctest: +SKIP
```

Comment:
Same as above, this doctest runs fine for me and might not need to be skipped. Also, to be consistent with the doctests for `DataFrame.nunique()`, `Series([1, 2, 3, np.nan])` should probably be moved to its own variable at the beginning of the doctests.